integration-test/src/test/java/com/datastrato/gravitino/integration/test/web/ui/CatalogsPageTest.java

```diff
@@ -109,70 +254,130 @@ public void testViewTabMetalakeDetails() throws InterruptedException {
+    String mysqlNode =
+        String.format("{{%s}}{{%s}}{{%s}}", METALAKE_NAME, MYSQL_CATALOG_NAME, CATALOG_TYPE);
+    catalogsPage.clickTreeNode(mysqlNode);
+    Assertions.assertTrue(catalogsPage.verifyGetCatalog(MYSQL_CATALOG_NAME));
+    String pgNode =
+        String.format("{{%s}}{{%s}}{{%s}}", METALAKE_NAME, PG_CATALOG_NAME, CATALOG_TYPE);
+    catalogsPage.clickTreeNode(pgNode);
+    Assertions.assertTrue(catalogsPage.verifyGetCatalog(PG_CATALOG_NAME));
+    String filesetNode =
+        String.format("{{%s}}{{%s}}{{%s}}", METALAKE_NAME, FILESET_CATALOG_NAME, "fileset");
+    catalogsPage.clickTreeNode(filesetNode);
+    Assertions.assertTrue(catalogsPage.verifyGetCatalog(FILESET_CATALOG_NAME));
+    String hiveNode =
+        String.format("{{%s}}{{%s}}{{%s}}", METALAKE_NAME, MODIFIED_CATALOG_NAME, CATALOG_TYPE);
+    catalogsPage.clickTreeNode(hiveNode);
+    Assertions.assertTrue(catalogsPage.verifyShowTableTitle("Schemas"));
+    Assertions.assertTrue(catalogsPage.verifyGetCatalog(MODIFIED_CATALOG_NAME));
+    String schemaNode =
+        String.format(
+            "{{%s}}{{%s}}{{%s}}{{%s}}",
+            METALAKE_NAME, MODIFIED_CATALOG_NAME, CATALOG_TYPE, SCHEMA_NAME);
+    catalogsPage.clickTreeNode(schemaNode);
+    Assertions.assertTrue(catalogsPage.verifyShowTableTitle("Tables"));
+    Assertions.assertTrue(catalogsPage.verifyShowDataItemInList(TABLE_NAME));
+    String tableNode =
+        String.format(
+            "{{%s}}{{%s}}{{%s}}{{%s}}{{%s}}",
+            METALAKE_NAME, MODIFIED_CATALOG_NAME, CATALOG_TYPE, SCHEMA_NAME, TABLE_NAME);
+    catalogsPage.clickTreeNode(tableNode);
+    Assertions.assertTrue(catalogsPage.verifyShowTableTitle("Columns"));
+    Assertions.assertTrue(catalogsPage.verifyShowDataItemInList(COLUMN_NAME));
+    Assertions.assertTrue(catalogsPage.verifyTableColumns());
   }
 
   @Test
-  @Order(11)
+  @Order(15)
+  public void testTreeNodeRefresh() throws InterruptedException {
+    createTableAndColumn(
+        METALAKE_NAME, MODIFIED_CATALOG_NAME, SCHEMA_NAME, TABLE_NAME_2, COLUMN_NAME_2);
+    String hiveNode =
+        String.format("{{%s}}{{%s}}{{%s}}", METALAKE_NAME, MODIFIED_CATALOG_NAME, CATALOG_TYPE);
+    catalogsPage.clickTreeNode(hiveNode);
+    String schemaNode =
+        String.format(
+            "{{%s}}{{%s}}{{%s}}{{%s}}",
+            METALAKE_NAME, MODIFIED_CATALOG_NAME, CATALOG_TYPE, SCHEMA_NAME);
+    catalogsPage.clickTreeNodeRefresh(schemaNode);
+    String tableNode =
+        String.format(
+            "{{%s}}{{%s}}{{%s}}{{%s}}{{%s}}",
+            METALAKE_NAME, MODIFIED_CATALOG_NAME, CATALOG_TYPE, SCHEMA_NAME, TABLE_NAME_2);
+    catalogsPage.clickTreeNode(tableNode);
+    Assertions.assertTrue(catalogsPage.verifyShowTableTitle("Columns"));
+    Assertions.assertTrue(catalogsPage.verifyShowDataItemInList(COLUMN_NAME_2));
+    Assertions.assertTrue(catalogsPage.verifyTableColumns());
+  }
+
```

Comment:
IMO, should add a `testRefreshPage` step here, and verify selected tree node each catalog level, schema level and table level. And make sure the expended tree node is correct, and verify table values too.